### ISSUE TITLE
Preparation of ScanNet

### ISSUE THREAD

**Issue by Ze-Yang** [facebookresearch/DepthContrast#12]
Hi, could you provide more detailed instructions about how to prepare the ScanNet dataset for model training? I have downloaded the ScanNet dataset but found it hard to follow the steps you provide, e.g., how to specify the arguments in the command below.
`python extract_pointcloud.py /path/to/extracted_data /path/to/extracted_pointcloud_visualization /path/to/extracted_pointclouds scannet_datalist.npy`

**Comment by zaiweizhang** [comment]
Sorry for the confusion. The previous documentation does provide the full details. 
I have updated the readme and some new codes. Please take a look!

**Comment by zaiweizhang** [comment]
Closing for now. Feel free to re-open it for future questions.
